lua/metals/config.lua

```diff
@@ -75,7 +75,13 @@ end
 local function check_for_coursier()
   -- this must be first the second "cs" is also found as binary but we should be more specific on Windows
   if util.is_windows then
-    return "cs.bat"
+    if util.has_bins("cs.bat") then
+      return "cs.bat"
+    else
+      log.error_and_show("Warning:Could not find cs.bat in PATH")
```

Comment:
Let's combine these three into one call, and put it in `messages.lua`.  You can see some other example there. I believe to see the `/` you just need to escape it. Can you try with `\\`?

Reply:
> Let's combine these three into one call, and put it in `messages.lua`. You can see some other example there.

Ok, I see that you already have a message for when coursier is not installed. 
```M.coursier_not_installed = [[
It looks like you don't have Coursier installed, which you need to install Metals.

You can find instructions on how to install it here: https://get-coursier.io/docs/cli-installation]]
```
Do you want me to use this one or to create another one more specific for Windows. I think another one would be better since the instructions on coursier's website does not mention to add coursier to the path. I could also modify this one instead to be more specific for windows.

> I believe to see the `/` you just need to escape it. Can you try with `\\`?

I tried it first like this `C:\\Users\\YOURNAME\\AppData\\Coursier\\data\\bin` but the backslashes would not appear in the final string, it gave me the string  `C:UsersMynameAppDataCoursierdata^Hin `.

Reply:
I'd say having a windows specific message here may be best. So we can detect the OS and then show then the windows one and for everything else we an use the existing one.

As for the path, that's really odd. I'm not sure why that's happening. I'd honestly rather keep that part out then potentially confuse people by putting `/path` format with windows.

Reply:
> I'd say having a windows specific message here may be best. So we can detect the OS and then show then the windows one and for everything else we an use the existing one.
> 
Ok, here's a proposal:
```
Warning: Could not find cs.bat in PATH
You can find instructions on how to install it here: https://get-coursier.io/docs/cli-installation#windows
Make sure to run cs.exe once to install Coursier and add it's installation directory to your path. 
See the note in the Readme.
```
> As for the path, that's really odd. I'm not sure why that's happening. 

Yes me neither. I tried `:lua print("C:\\Users\\YOURNAME\\AppData\\Coursier\\data\\bin")` and it escapes everything properly. 

>I'd honestly rather keep that part out then potentially confuse people by putting `/path` format with windows.

Ok, I'll just make a reference to the Readme.